src/main/resources/sprache/English.properties

```diff
@@ -2276,3 +2276,158 @@ ls.copyright.authoroftheformulas=The author of the formulas
 ls.transfer.motherclubfee=Motherclub fee income
 ls.transfer.previousclubfee=Previous club fee income
 
+Reference=Reference
+Value=Value
+Default=Default
+
+ls.color.black=Black
+ls.color.white=White
+ls.color.red=Red
+ls.color.blue=Blue
+ls.color.green=Green
+ls.color.orange=Orange
+ls.color.dark_gray=Dark gray
+ls.color.light_gray=Light gray
+ls.color.lightgreen=Light green
+ls.color.lightyellow=Light yellow
+ls.color.ho_gray1=Extra light gray
+ls.color.forest_green=Forest green
+ls.color.gray=Gray
+ls.color.yellow=Yellow
+ls.color.url_panel_bg=URL font color
+ls.color.panel_bg=Panel background
+ls.color.panel_border=Panel border
+ls.color.player_position_panel_border=Player position panel border
+ls.color.label_fg=Label font color
+ls.color.label_error_fg=Error label font color
+ls.color.list_fg=List font color
+ls.color.list_selection_bg=List selection background
+ls.color.button_bg=Button background
+ls.color.button_assist_cancel_bg=Assistant cancel button
+ls.color.button_assist_ok_bg=Assistant OK button
+ls.color.table_selection_bg=Table selection
+ls.color.table_selection_fg=Table selection font color
+ls.color.tableentry_bg=Table entry
+ls.color.tableentry_fg=Table entry font color
+ls.color.background_container=Container background
+ls.color.tableentry_improvement_fg=Table improvement entry font color
+ls.color.tableentry_decline_fg=Table decline entry font color
+ls.color.rating_border_below_limit=Rating border below limit
+ls.color.rating_border_above_limit=Rating border above limit
+ls.color.skillentry2_bg=2nd skill entry font color
+ls.color.player_pos_bg=Player position background
+ls.color.player_subpos_bg=Player position behaviour
```

Comment:
US English seems to be the language of choice (cf. “color” & “gray”) so for consistency this should be “behavior.”

Reply:
thanks - learned again.
But i see, that there are a lot of other cases, where this is wrong...

Reply:
Very possibly, I think this hasn't been super-consistent over the years.  Maybe we need to decide which one we pick, and create a ticket to update (I can take care of it).  My preference would be UK english, being in Ireland and all, and making more sense in the context of football.

Reply:
Now I got it